src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/view-alert-conditions.mdx

```diff
@@ -0,0 +1,342 @@
+
+We're going to customize these advanced signal settings for our condition that is montioring `pageviews` in the `WebPortal` application. 
+
+<img
+    title="fine tune alert settings"
+    alt="A screenshot depicting advanced signal settings"
+    src={alertsFineTuneAlertSignals}
+/>
+
+<CollapserGroup>
+  <Collapser
+    id="window-duration"
+    title="Window duration"
+  >
+
+  Setting the window duration for your alert condition tells New Relic how to group your data. If you're creating an alert condition for a data set that sends a signal to New Relic once every hour, you'd want to set the window duration to something closer to sixty minutes because it'll help spot patterns and unusual behavior. But, if you're creating an alert condition to monitor `pageviews` on your `WebPortal` application and New Relic collects a signal for that data every minute, we'd recommend setting the window duration to one minute. 
+  
+  When you're getting started we recommend sticking with our default settings, but the more you get familiar with creating an alert condition we encourage you to customize these fields based on your own experience. 
+  </Collapser>
+
+  <Collapser
+    id="sliding-window"
+    title="Use sliding window aggregation"
+  >
+
+  Throughout the day, data streams from your application into New Relic. Instead of evaluating that data immediately for incidents, alert conditions collect the data over a period of time known as the **aggregation window**. An additional delay allows for slower data points to arrive before the window is aggregated.
+
+  Sliding windows are helpful when you need to smooth out "spiky" charts. One common use case is to use sliding windows to smooth line graphs that have a lot of variation over short periods of time in cases where the rolling aggregate is more important than aggregates from narrow windows of time.
+
+  We recommend using our sliding window aggregation if you're not expecting to have a steady and consistent stream of data but are expecting some dips and spikes in data. 
+
+  You can learn more about sliding window aggregation in [this NRQL tutorial](/docs/query-your-data/nrql-new-relic-query-language/nrql-query-tutorials/create-smoother-charts-with-sliding-windows/) or by watching this video.
+
+  <Video
+    type="youtube"
+    id="-5--8DZynFE"
+  />
+
+  </Collapser>
+  
+  <Collapser
+    id="streaming-method"
+    title="Streaming method"
+  >
+  In general, we recommend using the **event flow** streaming method. This is best for data that comes into your system frequently and steadily. There are specific cases where **event timer** might be a better method to choose, but for your first alert we recommend our default, **event flow**. To better understand which streaming method to choose, we recommend watching this brief video.  
```

Comment:
I see that you're building story inside the collapser. I think it's good content, but per last comment, the section needs a bit more "set up" before you get into the meaty stuff here